core/render/xml/__init__.py

```diff
@@ -1,47 +1,34 @@
 from .default import DefaultRender as default, serializeXML
 from .user import UserRender as user
-from viur.core import conf
-from viur.core import securitykey
+from viur.core import conf, securitykey, exposed
 import datetime
 
 __all__ = [default]
 
 
+@exposed
 def genSkey(*args, **kwargs):
     return "<securityKey>%s</securityKey>" % securitykey.create()
 
 
-genSkey.exposed = True
-
-
+@exposed
 def timestamp(*args, **kwargs):
     d = datetime.datetime.now()
     return serializeXML(d.strftime("%Y-%m-%dT%H-%M-%S"))
 
 
-timestamp.exposed = True
-
-
-def generateAdminConfig(adminTree):
+@exposed
+def dumpConfig(adminTree):
     res = {}
-    for key in dir(adminTree):
+    for key in dir(conf["viur.mainApp"].xml):
```

Comment:
This should be the same as in the vi renderer:
```py
    for key in dir(conf["viur.mainApp"].xml):
        module = getattr(conf["viur.mainApp"].xml, key)
        if not isinstance(module, Module):
            continue
        if admin_info := module.describe():
            res[key] = admin_info

    res = {
        "modules": res,
        "configuration": {k.removeprefix("admin."): v for k, v in conf.items() if
                          k.lower().startswith("admin.")}
    }
    return res
```

Reply:
tbd

Reply:
fixed.